tools/github/scripts/Validate-AzsdkCodeOwner.ps1

```diff
@@ -16,8 +16,8 @@ $hasPermissions = $false
 $orgResponse = (gh api "https://api.github.com/users/$UserName/orgs")
 $orgs = $orgResponse | ConvertFrom-Json
 
-if ($orgs -ne $null) {
-    $orgs = $orgs | select -Expand login
+if (!$org) {
+    $orgs = $orgs | Select-Object -Expand login
```

Comment:
```suggestion
    try {
        $orgs = $orgs | Select-Object -Expand login
    } catch {
        $orgs = @()
   }
```

Since you're making a change anyway, I ran into an unexpected edge case today where the user was valid and there was a payload but it did not contain a login.

Reply:
Better to use `-ErrorAction Ignore` in that case.

Reply:
wouldn't that cause `$orgs` to be null, then?

Reply:
Using exceptions for conditional logic leads down the highway to the danger zone.